### PR TITLE
Automaps the book of kindred

### PR DESCRIPTION
**Pull request by Arturlang**
## About The Pull Request
Makes the map changes by the previous bloodsucker PR automapped. Also removes a unused var.
And some backup ways for the book of kindred to spawn if all else fails

## Why It's Good For The Game
Deals with conflicts better, or so i'm told

## Proof Of Testing
I went through every map and checked it. Really don't want to post 10 or so pics.

## Changelog
:cl:
add: Adds backup ways for the book of kindred to spawn, either spawning on a curator or on the codex gigas
/:cl:

### CODE REVIEW

File: _maps/skyrat/automapper/automapper_config.toml [hunk 1]
```diff
@@ -459,3 +459,102 @@ directory = "_maps/bubber/automapper/templates/generic/"
...
+coordinates = [49, 103, 2]
+trait_name = "Station"
+
+#BurgerStation
+[templates.burgerstation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "burgerstation.dmm"
+coordinates = [75, 117, 1]
+trait_name = "Station"
```
Comment (StrangeWeirdKitten) [inline _maps/skyrat/automapper/automapper_config.toml:552]: We own this map, and this map is deleted on the codebase. Moonstation is the updated version but it's not merged yet, so I'll ask Burger to update moonstation with it.

File: _maps/skyrat/automapper/automapper_config.toml [hunk 2]
```diff
@@ -459,3 +459,102 @@ directory = "_maps/bubber/automapper/templates/generic/"
...
+coordinates = [175, 140, 3]
+trait_name = "Station"
+
+#LimaStation
+[templates.limastation_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "LimaStation.dmm"
+coordinates = [99, 128, 2]
+trait_name = "Station"
```
Comment (StrangeWeirdKitten) [inline _maps/skyrat/automapper/automapper_config.toml:488]: Depreciated map we also own. Not needed to automap it, but we also don't maintain it anymore

File: _maps/skyrat/automapper/automapper_config.toml [hunk 3]
```diff
@@ -459,3 +459,102 @@ directory = "_maps/bubber/automapper/templates/generic/"
...
+coordinates = [146, 137, 1]
+trait_name = "Station"
+
+#Biodome
+[templates.biodome_book_of_kindred]
+map_files = ["book_of_kindred_display.dmm"]
+directory = "_maps/bubber/automapper/templates/generic/"
+required_map = "biodome.dmm"
+coordinates = [116, 102, 1]
+trait_name = "Station"
```
Comment (StrangeWeirdKitten) [inline _maps/skyrat/automapper/automapper_config.toml:536]: Not needed either. This can be directly mapped

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
We own these maps and you don't need to automap it. Also burgerstation is removed from the codebase

File: _maps/skyrat/automapper/automapper_config.toml (hunk 1 above)
Comment (Arturlang) [reply _maps/skyrat/automapper/automapper_config.toml:552]: Was just more neat to have all the related map changes in one thing, but I'll revert the changes for these maps

**Comment by Arturlang** [comment]
Any other map changes besides the kindred stuff is due to strongdmm cleaning up duplicate variables

File: _maps/skyrat/automapper/automapper_config.toml (hunk 1 above)
Comment (StrangeWeirdKitten) [reply _maps/skyrat/automapper/automapper_config.toml]: Yeah I get it, but I want to limit the automapping on things we don't need it for. If we need some level of enforced consistency, then a mapping spawner can be coded in where you can change the contents without touching map files

**Comment by Arturlang** [comment]
PR's ready

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Resolve the mapping conflicts and I can merge this

**Comment by Arturlang** [comment]
> Resolve the mapping conflicts and I can merge this

done

**Pull request merged**